helm/charts/Chart.yaml

```diff
@@ -0,0 +1,8 @@
+---
+apiVersion: v2
+name: charts
+description: A Helm chart for platform-health
+
+type: application
+version: 0.1.0
+appVersion: 0.3.0
```

Comment:
Would it be feasible to update this dynamically in the `release` workflow to match the tag being built, prior to build?
Should be as simple as `sed -iP "s/^\(version:\) .*/\1 ${GITHUB_REF#refs/tags/v}/" helm/charts/Chart.yaml` before the `helm template …` call.
This then negates the need to explicitly pass a `--version` on the `helm update …` call.

Reply:
Simpler with a `--version ${{ github.ref }}`. ;-)